aries-site/src/examples/components/layer/LayerFormExample.js

```diff
@@ -0,0 +1,173 @@
+              align="center"
+              as="header"
+              justify="between"
+            >
+              <Heading margin="none" level={2} size="small">
+                Add Monitor
+              </Heading>
+            </Header>
+            <Box
+              // Padding used to prevent focus from being cutoff
+              pad={{ horizontal: 'xsmall' }}
+              // overflow to allow the body to scroll with keeping the
+              // header at the top
+              overflow="auto"
+            >
+              <Form
+                onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+                value={formValues}
+                onChange={onFormChange}
+                method="post"
+              >
+                <Box margin={{ bottom: 'small' }}>
+                  <FormField name="monitorName" label="Monitor Name">
+                    <TextInput name="monitorName" />
+                  </FormField>
+                  <FormField name="desc" label="Bussiness Description">
+                    <TextArea name="monitorName" />
+                  </FormField>
+                  <Text
+                    margin={{ vertical: 'medium' }}
+                    weight="bold"
+                    color="text-strong"
+                  >
+                    Monitor Detail
+                  </Text>
+                  <FormField name="monitorType" label="Monitor Type">
+                    <TextInput name="monitorType" />
+                  </FormField>
+                  <FormField name="rules">
+                    <CheckBox
+                      name="rules"
+                      label="Include Process Rules"
+                      toggle
+                      fill
+                      checked
+                      reverse
+                    />
+                  </FormField>
+                  <FormField name="frameworks" label="Frameworks">
+                    <Select options={options} name="frameworks" />
+                  </FormField>
+                  <FormField name="severity" label="Rule Severity">
+                    <Select options={options} name="severity" />
+                  </FormField>
+                  <FormField name="provider" label="Provider">
+                    <Select options={options} name="provider" />
+                  </FormField>
+                  <FormField name="account" label="Account">
+                    <Select options={options} name="account" />
+                  </FormField>
+                  <FormField name="ruleName" label="Rule Name">
+                    <Select options={options} name="ruleName" />
+                  </FormField>
+                  <Text
+                    margin={{ vertical: 'medium' }}
+                    weight="bold"
+                    color="text-strong"
+                  >
+                    Email Notification
+                  </Text>
+                  <FormField name="notifications">
```

Comment:
These FormFields and inputs are missing `htmlFor` and `id` attributes.

Reply:
I added all of these and put the Form in a separate file to reuse